android/src/ti/styledlabel/parsing/HtmlToSpannedConverter.java

```diff
@@ -498,16 +498,42 @@ private void translateKeyValueInToSpannable(TagMarker tm, String key, String val
 
 	private void startImg(Attributes attributes) {
 		String src = attributes.getValue("", "src");
+		String width= attributes.getValue("", "width");
+		String height= attributes.getValue("", "height");
 		Drawable d = null;
 
 		if (mImageGetter != null) {
 			d = mImageGetter.getDrawable(src);
 		}
 
+		
 		if (d == null) {
 			mSB.setSpan(new BackgroundColorSpan(0), mSB.length(), mSB.length(), Spannable.SPAN_EXCLUSIVE_EXCLUSIVE);
 			return;
 		}
+		else{
+			int weidthInAttributes=d.getIntrinsicWidth();
+			int heightInAttributes=d.getIntrinsicHeight();
+			if(width!=null){
+				try{
+					weidthInAttributes=Integer.parseInt(width);
+				}
+				catch(NumberFormatException nfe){
+					Util.e("NumberFormatException on width attribute");
+				}
+			}
+			if(height!=null){
+				try{
+					heightInAttributes=Integer.parseInt(height);
+				}
+				catch(NumberFormatException nfe){
+					Util.e("NumberFormatException on height attribute");
+				}
+			}
+			d.setBounds(0,0,weidthInAttributes,heightInAttributes);
+		}
+		
```

Comment:
Spaces all over the place. Please ensure to make one space in `if () {` and `else  {`, also put the else in the same line `} else {`. Also space between all other brackets like try, catch and comparators.